build/params_2k.go

```diff
@@ -13,8 +13,9 @@ const BreezeGasTampingDuration = 0
 
 const UpgradeSmokeHeight = -1
 const UpgradeIgnitionHeight = -2
-const UpgradeLiftoffHeight = -3
+const UpgradeRefuelHeight = -3
 const UpgradeActorsV2Height = 10
+const UpgradeLiftoffHeight = -4
```

Comment:
I notice liftoff changed from -3 to -4 -- does that need to change anywhere else?

Reply:
These can be any negative number (this file is only built for testing).

Reply:
The other place would be build/params_testground.go, which is already done